theano/gpuarray/tests/GpuArray.pkl

```diff
@@ -1,4 +1,4 @@
-ctheano.sandbox.gpuarray.type
```

Comment:
Should we add something for backward compat?  Otherwise unfixed pickles will break.